route_test.go

```diff
@@ -488,40 +488,6 @@ func assertHttpNormal(t *testing.T, addr string, expectNormal bool) {
 	}
 }
 
-func longMiddleware(_ config.Config) contractshttp.Middleware {
-	return func(ctx contractshttp.Context) {
-		time.Sleep(2 * time.Second)
-		ctx.Request().Next()
-	}
-}
-
-func TestTimeoutMiddlewareIntegration(t *testing.T) {
```

Comment:
Sorry, I made a mistake comment before, the timeout middleware test case doesn't need to be added in `route_test.go`, use `middleware_timeout_test.go` instead.